10-Modelling-Deformable-Interfaces-with-Meshes/02-Vesicle-MD.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>Does the volume need to be slowly changed here? What if you just abruptly set the new volume and run the simulation? I think <code>Brownian</code> would be able to absorb the energy, but maybe not. Taking out the custom action would make the tutorial easier for new users to understand. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>